src/lakefs_spec/client_helpers.py

```diff
@@ -26,6 +35,25 @@ def commit(
     message: str,
     metadata: dict[str, str] | None = None,
 ) -> Commit:
+    """Creates a new commit of all uncommitted changes on the branch in the lakeFS file storage
+
+    Parameters
+    -------
+    client: LakeFSClient
+        lakeFS client object.
+    repository: str
+        Repository name.
+    branch: str
+        Branch name.
+    message: str
+        Commit message.
+    metadata: dict[str, str] | None, optional
+        Additional metadata for the commit. Defaults to None.
```

Comment:
```suggestion
        Additional metadata to include in the commit.
```

(The default is clear from the signature.)